scripts/docker/docker-compose.yml

```diff
@@ -95,6 +95,8 @@ services:
     image: azureiotpcs/pcs-config-java:latest
     depends_on:
       - storageadapter
+      - devicesimulation
```

Comment:
isn't this a circular dependency?

Reply:
No. deviceSimulation doesn't depend on config service.